migtests/scripts/live-migration-run-test.sh

```diff
@@ -126,7 +127,7 @@ main() {
 	step "Archive Changes."
 	archive_changes &
 
-	sleep 30 
+	sleep 60 
```

Comment:
Is this for any reason in normal and fallback and not in fall-forward?

Reply:
Fall Forward also has this but after trapping the archive changes process. It's set to 120 currently. Let me change to 60.